egui_glow/src/painter.rs

```diff
@@ -35,6 +35,8 @@ pub struct Painter {
     is_embedded: bool,
     vertex_array: crate::misc_util::VAO,
     srgb_support: bool,
+    /// The filter used for subsequent textures.
+    texture_filter: u32,
```

Comment:
wouldn't it be nicer to use `TextureFilter` here and defer the call to `glow_code`? It would keep the code typesafe for longer.

```suggestion
    texture_filter: TextureFilter,
```

Reply:
Sure, seems reasonable to me!